packaging/generic-deb/debian/rules

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/make -f
+# -*- makefile -*-
+VERSION := $(shell git describe --tags | sed -e 's/v//' -e 's/-.*//')
```

Comment:
Assuming that you're building in a git working copy will cause problems. The source artifacts that you're generating in the build process (the .orig.tar.* and *.dsc files) won't contain the git metadata, and thus it won't be possible to use them to generate binaries.

Since you've got version information in `debian/changelog` by the time this runs, you should use `dpkg-parsechangelog -S Version` in place of `git describe --tags`

Reply:
changed to reading from a text file next to the agent tarball

Reply:
using dpkg-parsechangelog now